api/scripts/generate-certification-csv-results-by-session-ids.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+require('dotenv').config({ path: `${__dirname}/../.env` });
+
+const fs = require('fs');
+const bluebird = require('bluebird');
+const isEmpty = require('lodash/isEmpty');
+const logger = require('../lib/infrastructure/logger');
+const certificationResultUtils = require('../lib/infrastructure/utils/csv/certification-results');
+const usecases = require('../lib/domain/usecases');
+
+/**
+ * Avant de lancer le script, remplacer la variable DATABASE_URL par l'url de la base de réplication
+ * Usage: NODE_TLS_REJECT_UNAUTHORIZED='0' PGSSLMODE=require node scripts/generate-certification-csv-results-by-session-ids.js 1234,5678,9012
+ */
+async function main() {
+  logger.info('Début du script de génération de résultats pour une session.');
+
+  if (process.argv.length <= 2) {
+    logger.info(
+      'Usage: NODE_TLS_REJECT_UNAUTHORIZED="0" PGSSLMODE=require node scripts/generate-certification-csv-results-by-session-ids.js 1234,5678,9012'
+    );
+    return;
+  }
+
+  const sessionIds = process.argv[2].split(',');
+
+  await bluebird.mapSeries(sessionIds, async (sessionId) => {
+    const { session, certificationResults } = await usecases.getSessionResults({ sessionId });
+
+    if (isEmpty(certificationResults)) {
+      throw new Error(`Pas de résultat trouvé pour la session ${sessionId}`);
+    }
+
+    const csvResult = await certificationResultUtils.getSessionCertificationResultsCsv({
+      session,
+      certificationResults,
+    });
+
+    logger.info(`Génération du fichier CSV.`);
+
+    // eslint-disable-next-line no-sync
+    fs.writeFileSync(`resultats-session-${sessionId}.csv`, csvResult);
```

Comment:
Ne pas utiliser les methodes filesystem synchrone qui vont bloquer node
```suggestion
    await fs.promises.writeFile(`resultats-session-${sessionId}.csv`, csvResult);
```